src/bmark_solvers.jl

```diff
@@ -16,10 +22,27 @@ Any keyword argument accepted by `solve_problems`
 A Dict{Symbol, AbstractExecutionStats} of statistics.
 """
 function bmark_solvers(solvers::Dict{Symbol, <:Any}, args...; kwargs...)
-  stats = Dict{Symbol, DataFrame}()
-  for (name, solver) in solvers
-    @info "running solver $name"
-    stats[name] = solve_problems(solver, name, args...; kwargs...)
-  end
-  return stats
-end
+    tasks = Vector{Task}(undef, length(solvers))
+    names = Vector{Symbol}(undef, length(solvers))
+    idx = 0
+    for (name, solver) in solvers
+        idx += 1
+        names[idx] = name
+        # Make a deep copy of the problems for this task so the race condition doesn't occur
+        # local_args = deepcopy.(args)
+        local_args = args
+        local_solver = deepcopy(solver)
```

Comment:
I'm not sure I understand what this is. At this point, the solver has not yet been instantiated, right? So this is just a copy of a function?!